docs/rules/better-regex.md

```diff
@@ -1,11 +1,10 @@
 # Improve regexes by making them shorter, consistent, and safer
 
-<!-- Do not manually modify RULE_NOTICE part. Run: `npm run generate-rule-notices` -->
-<!-- RULE_NOTICE -->
-✅ *This rule is part of the [recommended](https://github.com/sindresorhus/eslint-plugin-unicorn#recommended-config) config.*
+✅ This rule is enabled in the `recommended` [config](https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs).
```

Comment:
```suggestion
✅ This rule is enabled in the [`recommended` config](https://github.com/sindresorhus/eslint-plugin-unicorn#preset-configs).
```

THis feels more correct. The link is for the "recommended config".

Reply:
Agree with the other suggestions, but for this one, I currently only have a [`--url-configs` option](https://github.com/bmish/eslint-doc-generator#configuration-options) for specifying a link to general documentation about configurations in the plugin, and not the `recommended` config specifically. So that's why I put the link on the `config` word only. Do you still think it makes sense to include the config name in the link, even though the link isn't specifically pointing to that configuration?

Reply:
I missed that it was not pointing at a specific config. You're right. Doesn't make sense.